docs/manual/developer_guide.adoc

```diff
@@ -10,6 +10,10 @@ toc::[]
 
 == Introduction
 
+This document tries provides information useful for ComplianceAsCode/content project developers.
```

Comment:
Slight reword: This document provides information useful for `ComplianceAsCode/content` project contributors.